internal/controller/controllers/infraenv_controller.go

```diff
@@ -167,6 +167,9 @@ func (r *InfraEnvReconciler) updateInfraEnv(ctx context.Context, log logrus.Fiel
 	if infraEnv.Spec.SSHAuthorizedKey != internalInfraEnv.SSHAuthorizedKey {
 		updateParams.InfraEnvUpdateParams.SSHAuthorizedKey = &infraEnv.Spec.SSHAuthorizedKey
 	}
+	if infraEnv.Spec.AdditionalTrustBundle != internalInfraEnv.AdditionalTrustBundle {
```

Comment:
Thinking about this a bit more ... if we trim this for the user on the way in won't this always be true if the added it with a space?

Is that a problem? Wondering if this could cause a hot loop.

Reply:
If the only way that this value can be set is through the registerInfraEnvInteral and updateInfraEnvInternal methods and these both perform the trim operation then I don't see the problem?

Or am I mistaken?

Reply:
The value in the infraenv spec will continue to have the space and the stored value will be updated.